pkg/addonmanager/manager.go

```diff
@@ -153,6 +167,13 @@ func (a *addonManager) Start(ctx context.Context) error {
 		a.addonAgents,
 	)
 
+	addonConfigController := addonconfig.NewAddonConfigController(
```

Comment:
nit: let's check if there is configs and start this part. I think we should have a separated start func  to start it.